### ISSUE TITLE
tmux fails: `open terminal failed: can't find terminfo database`

### ISSUE THREAD

**Issue by timotheecour** [tmux/tmux#743]
tmux
open terminal failed: can't find terminfo database

echo $TERM
xterm-256color

these are non-empty:

/etc/terminfo
/lib/terminfo
/usr/share/terminfo

**Comment by nicm** [comment]
This issue does not include enough information. Please read the [CONTRIBUTING](https://github.com/tmux/tmux/blob/master/CONTRIBUTING) file.

**Comment by timotheecour** [comment]
Sure (maybe use a github issue template? would be more obvious)
Ubuntu 16.04.1, tmux 2.3

tmux -Ltest kill-server
tmux -vvvv -Ltest -f/dev/null new
did work

had to `tmux kill-server` and then it worked; looks like my existing session was corrupt for some reason. closing

**Comment by timotheecour** [comment]
happened again. Would be good to have a workaround that wouldn't involve killing existing sessions

**Comment by nicm** [comment]
try running with "strace -fostrace.out tmux" and send me the strace.out file

**Comment by timotheecour** [comment]
Ok, next time this happens (already killed session)

**Comment by karandwivedi42** [comment]
I face the same issue. Here is my strace.out

https://gist.github.com/karandwivedi42/692e5dc55bf880b3c1523554d2768884

**Comment by jsatk** [comment]
I'm also having this issue on a brand new computer with tmux installed from homebrew.

**Comment by bemeurer** [comment]
Same here, got the computer out of the box today and installed tmux from homebrew.

**Comment by deadlysyn** [comment]
for anyone finding this via google after a brew/ncurses update... all i needed was rm -rf /tmp/tmux* and then tmux fired up just fine. running both of the commands above alone didn't do it for me on macos/iterm/latest brew tmux. moving to other TERM types didn't either, and would be less than satisfactory. hoping this simple fix will work for others.

**Comment by benw01** [comment]
For anyone finding this on google: I had a similar issue. When I detached from my running tmux session, I couldn't subsequently re-attach, with the same error message: `open terminal failed: can't find terminfo database`.

* MacOS Mojave (10.14.6)
* tmux 3.0a
* Terminal.app

Turns out homebrew had upgraded ncurses (from 6.1_1 to 6.2) from underneath my running tmux server process, and deleted the folder for the previous ncurses under `/usr/local/Cellar/ncurses`.

I restored ` /usr/local/Cellar/ncurses/6.1_1` from a backup, and was then able to `tmux att` again. No need to kill the running server or remove the tmux socket in `/tmp`.
